.github/workflows/main.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [12.x, 14.x]
+        node-version: [14.x, 16.x, 18.x]
```

Comment:
I don't think we need 16. Let's just do 14 and 18

Reply:
I couldn't figure out why I didn't need 16 at first glance, why?

Reply:
Why we don't need 16? Just because it shouldn't be of issue.

Generally we'd want to support the oldest LTS (14) and newest that will become an LTS (18).

16 shouldn't cause issues so it's not worth running against as having it will just increase overall CI run time.

Reply:
I see, I was trying to check all currently supported versions, but that makes sense. Thank you very much.
I will correct it.

Reply:
Fixed!! https://github.com/preactjs/wmr/pull/936/commits/58cbdf2accf064930560c7a3d9c855983c4f49e7